src/Services/Implementations/DefaultAlgoliaTaskLogger.cs

```diff
@@ -51,11 +50,37 @@ public void HandleEvent(TreeNode node, string eventName)
             }
         }
 
+        public void HandleEventAfter(TreeNode node, string eventName)
+        {
+            var treeProvider = new TreeProvider();
+            if (eventName != DocumentEvents.Delete.Name)
+            {
+                node = treeProvider.SelectSingleDocument(node.DocumentID, coupledData: true);
+            }
+            
+            foreach (var indexName in IndexStore.Instance.GetAll().Select(index => index.IndexName))
+            {
+                if (!node.IsIndexedByIndex(indexName))
+                {
+                    continue;
+                }
+
+                try
+                {
+                    var queueItem = new AlgoliaQueueItem(node, GetTaskType(node, eventName), indexName, node.ChangedColumns());
+                    AlgoliaQueueWorker.Current.EnqueueAlgoliaQueueItem(queueItem);
+                }
+                catch (InvalidOperationException ex)
+                {
+                    eventLogService.LogException(nameof(DefaultAlgoliaTaskLogger), nameof(HandleEvent), ex);
```

Comment:
Either use correct method nameof() or use some other descriptive event code.

Reply:
Changed this to a parameter of the private method